common/schemas/keyman-touch-layout/keyman-touch-layout.clean.spec.json

```diff
@@ -88,6 +93,13 @@
         "flick": { "$ref": "#/definitions/flick" },
         "multitap": { "$ref": "#/definitions/subkeys" }
       },
+      "anyOf": [
+        {"required": ["id"]},
+        {"required": ["sp"]},
+        {"required": ["sk"]},
+        {"required": ["flick"]},
+        {"required": ["multitap"]}
+      ],
```

Comment:
Are there cases where we may wish to support an id-less key that has subkeys, flicks, or multitaps?

Granted, I'm not super-great with schemas, but my impression of this part of the spec appears to permit such a case.

I know we permit id-less keys with `sp` properties - such as for a 'blank' space for row padding, so no complaints on that one.